### PR TITLE
Remove wait and spammy logs from full-nodes-1 tests

### PR DESCRIPTION
**Pull request by eclark0426**

### CODE REVIEW

File: docker/flexnet/tests/full-nodes-1.py [hunk 1]
```diff
@@ -54,18 +47,10 @@ def main():
...
-            time.sleep(5)
-            print(f'sender balance {client.get_account_balance(sender_account)}')
-            print(f'receiver balance {client.get_account_balance(receiver_account)}')
-
+            time.sleep(2)
             for other_client_name, other_client in clients.items():
-                try:
-                    other_client.wait_for_transaction_receipt(tx_hash)
-                except Exception as e:
-                    print(f'{other_client_name}: wait_for_transaction_receipt exception {e}')
```
Comment (michael-yxchen) [inline docker/flexnet/tests/full-nodes-1.py:68]: Printing in other places are fine. I just think we shouldn't print error here, as it's expected to fail

File: docker/flexnet/tests/full-nodes-1.py [hunk 2]
```diff
@@ -75,18 +60,10 @@ def main():
...
```
Comment (michael-yxchen) [inline docker/flexnet/tests/full-nodes-1.py:89]: and here

**Comment by eclark0426** [comment]
Will drop the second commit that removes the additional logs, then

**Comment by michael-yxchen** [comment]
Thanks

**Pull request merged**